xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/javascript.vm

```diff
@@ -199,6 +199,11 @@ require(['jquery', 'iscroll', 'drawer'], function($, IScroll) {
       $('#tmDrawerActivator').attr('aria-expanded', 'false');
     });
   });
+    // Drawer can be closed by pressing the ESC key, irrespective of how you opened it (by using keyboard or by clicking it)
+    $(document).on('keydown', function (event) {
+      if (event.key === 'Escape') {
+        $('.drawer-nav').closest('body').drawer('close');}
+  });
```

Comment:
Still missing the formatting. If you don't, you should use an IDE which will do that automatically for you.

Reply:
Is it fine now?